pkgs/development/python-modules/torch/default.nix

```diff
@@ -1,5 +1,12 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, buildPythonPackage, python,
-  config, cudaSupport ? config.cudaSupport, cudaPackages, magma,
+  config, cudaSupport ? config.cudaSupport, cudaPackages,
+  effectiveMagma ?
+  if cudaSupport then magma-cuda-static
+  else if rocmSupport then magma-hip
+  else magma,
```

Comment:
See previous comment.

Reply:
Marking resolved as per https://github.com/NixOS/nixpkgs/pull/268746#discussion_r1400351241 and https://github.com/NixOS/nixpkgs/pull/268746#issuecomment-1820517722